src/components/TutorialWindowComponent.js

```diff
@@ -0,0 +1,102 @@
+import {
+  EventHandlerService,
+  PDFLEvents,
+} from "../services/EventHandlerService";
+/**
+ * Component representing tutorial window component and it's element
```

Comment:
This looks like unfinished sentence. You should maybe describe in more detail which tutorials can be displayed in this window (highlight, graph, summary)